addon/components/head-layout.js

```diff
@@ -0,0 +1,63 @@
+import Ember from 'ember';
+import layout from 'ember-cli-head/templates/components/head-layout';
```

Comment:
I'd generally prefer relative imports here, but I don't feel super strongly...

This would become:

```js
import layout from '../templates/components/head-layout';
```